analyzers/tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -64,6 +64,9 @@
     <None Remove="CBDEMocks\CBDESucceedsWithIncorrectResults.exe" />
     <None Remove="CBDEMocks\CBDEWaitAndSucceeds.exe" />
     <None Remove="ResourceTests\RuleWithBooleanParameter\SonarLint.xml" />
+    <None Remove="ResourceTests\SonarProjectConfig\Invalid_DifferentNamespace\SonarProjectConfig.xml" />
+    <None Remove="ResourceTests\SonarProjectConfig\Invalid_MixedSeparators\SonarProjectConfig.xml" />
+    <None Remove="ResourceTests\SonarProjectConfig\Invalid_DifferentClassName\SonarProjectConfig.xml" />
```

Comment:
I don't understand this piece:
Why are only 3 out of 13 files here? Is it intended? Does it have to be here? What's the reason for having 3 `None` and others as `Content`?